home.css

```diff
@@ -0,0 +1,27 @@
+body {
+    /* Visions of grandeur */
+    background: #000046;  /* fallback for old browsers */
+    background: -webkit-linear-gradient(to right, #000046, #1CB5E0) fixed;  /* Chrome 10-25, Safari 5.1-6 */
+    background: linear-gradient(to right, #000046, #1CB5E0) fixed; /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+    /*Have to put 'fixed' on end of linear-gradient for it to work / fill entire screen*/
+}
+
+.name {
+    text-align: center;
+    font-size: 3rem;
+    /*border: 1px solid #FFF;*/
```

Comment:
might be worth deleting unused code

Reply:
will do

Reply:
...but keeping in for now as am using it to check layout